src/components/partnership/partnership.repository.ts

```diff
@@ -124,50 +152,58 @@ export class PartnershipRepository extends DtoRepository<
-        relation('out', '', 'partner'),
-        node('partner', 'Partner'),
-        relation('out', '', 'organization', ACTIVE),
-        node('org', 'Organization'),
-      ])
-      .apply(matchPropsAndProjectSensAndScopedRoles(session, { view }))
-      .apply(matchChangesetAndChangedProps(view?.changeset))
-      .apply(matchProps({ nodeName: 'project', outputVar: 'projectProps' }))
-      .apply(
-        matchProps({
-          nodeName: 'project',
-          view,
-          optional: true,
-          outputVar: 'projectChangedProps',
-        }),
-      )
-      .return<{ dto: UnsecuredDto<Partnership> }>(
-        merge('props', 'changedProps', {
-          mouStart: coalesce(
-            'changedProps.mouStartOverride',
-            'props.mouStartOverride',
-            'projectChangedProps.mouStart',
-            'projectProps.mouStart',
-          ),
-          mouEnd: coalesce(
-            'changedProps.mouEndOverride',
-            'props.mouEndOverride',
-            'projectChangedProps.mouEnd',
-            'projectProps.mouEnd',
-          ),
-          parent: 'project',
-          project: 'project.id',
-          partner: 'partner.id',
-          organization: 'org.id',
-          changeset: 'changeset.id',
-          mou: { id: 'props.mou' },
-        }).as('dto'),
-      )
+      .apply(this.hydrate(session, view))
       .map('dto')
       .run();
   }
 
+  protected override hydrate(session: Session, view?: ObjectView) {
+    return (query: Query) =>
+      query
+        .match([
+          node('project'),
+          relation('out', '', 'partnership'),
+          node('node'),
+          relation('out', '', 'partner'),
+          node('partner', 'Partner'),
+          relation('out', '', 'organization', ACTIVE),
+          node('org', 'Organization'),
+        ])
+        .apply(matchPropsAndProjectSensAndScopedRoles(session, { view }))
+        .apply(matchChangesetAndChangedProps(view?.changeset))
+        .apply(matchProps({ nodeName: 'project', outputVar: 'projectProps' }))
+        .apply(
+          matchProps({
+            nodeName: 'project',
+            view,
+            optional: true,
+            outputVar: 'projectChangedProps',
+          }),
+        )
+        .return<{ dto: UnsecuredDto<Partnership> }>(
+          merge('props', 'changedProps', {
+            mouStart: coalesce(
+              'changedProps.mouStartOverride',
+              'props.mouStartOverride',
+              'projectChangedProps.mouStart',
+              'projectProps.mouStart',
+            ),
+            mouEnd: coalesce(
+              'changedProps.mouEndOverride',
+              'props.mouEndOverride',
+              'projectChangedProps.mouEnd',
+              'projectProps.mouEnd',
+            ),
+            parent: 'project',
+            project: 'project { .id }',
+            partner: 'partner { .id }',
+            organization: 'org { .id }',
+            changeset: 'changeset { .id }',
```

Comment:
We're still expecting the changeset to be an ID, not a object/link.
```suggestion
            changeset: 'changeset.id',
```

Reply:
Thank you @CarsonF!